app/presenters/contributions/index.rb

```diff
@@ -71,8 +72,20 @@ def contribution_table_row_data_cells(contribution)
         table_cell(contribution[:identifier].join('; ')),
         table_cell(contribution[:date]),
         table_cell(t(contribution[:status], scope: 'contribute.contributions.states')),
-        table_cell(contribution[:media] ? '✔' : '✘')
-      ]
+        table_cell(contribution[:media] ? '✔' : '✘'),
+        (@delete_buttons ? table_cell(contribution_delete_cell(contribution), row_link: false) : nil)
+      ].compact
+    end
+
+    def contribution_delete_cell(contribution)
+      if contribution[:status] == 'draft'
```

Comment:
We need to keep AASM the single source of truth for permitted state changes.

Perhaps, in the hashes built in `ContributionsController#assemble_index_contributions`, add a new element keyed `:deletable` with the return value of `::Contribution.new(aasm_state: contribution.aasm_state).wipeable?` and use that here.